docs/sphinx/users/micromanager/index.txt

```diff
@@ -0,0 +1,39 @@
+Micro-Manager
+=============
+
+- Micromanager offers the functionality for saving the acquired images in tiff/ome-tiff format. 
+- Based on the mode of saving and the configuration settings, the acquired image can be saved 
+  with/without a companion file (metadata.txt).
```

Comment:
Note the companion file will end in metadata.txt but will not be necessarily exactly `metadata.txt`